plugins/source/gcp/resources/services/dns/resource_record_sets.go

```diff
@@ -0,0 +1,28 @@
+package dns
+
+import (
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+	"github.com/cloudquery/plugins/source/gcp/client"
+	"google.golang.org/api/dns/v1"
+)
+
+func ResourceRecordSets() *schema.Table {
```

Comment:
```suggestion
func resourceRecordSets() *schema.Table {
```

You can make this private to indicate that it's a relational table and shouldn't be used directly